examples/gno.land/p/demo/nestedpkg/nestedpkg.gno

```diff
@@ -0,0 +1,84 @@
+// Package nestedpkg provides helpers for package-path based access control.
+// It is useful for upgrade patterns relying on namespaces.
+package nestedpkg
+
+import (
+	"std"
+	"strings"
+)
+
+// IsCallerSubPath checks if the caller realm is located in a subfolder of the current realm.
+func IsCallerSubPath() bool {
+	var (
+		cur  = std.CurrentRealm().PkgPath() + "/"
+		prev = std.PrevRealm().PkgPath() + "/"
```

Comment:
Instead of having `var` blocks in each of these functions, it is more typical in go to do a type inferred assignment using `:=`.

Reply:
Up to you, I like having horizontally aligned assignments with this pattern. But for only two, I'm fine just changing.